File: src/main/java/me/yoursole/ctf/Commands/Timer.java
package me.yoursole.ctf.Commands;

import me.yoursole.ctf.DataFiles.GameData;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;

public class Timer implements CommandExecutor {
    @Override
    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
        if (sender.isOp()) {
            if (args.length == 0) return false;
            if (args[0].equalsIgnoreCase("clear")) GameData.timerMs = -1L;
            else GameData.timerMs = System.currentTimeMillis() + Long.parseLong(args[0]);
        } else {
            return false;
        }
        return true;

    }
}

File: src/main/kotlin/me/yoursole/ctf/commands/Timer.kt
package me.yoursole.ctf.commands

import com.mojang.brigadier.CommandDispatcher
import com.mojang.brigadier.arguments.IntegerArgumentType
import me.yoursole.ctf.datafiles.GameData
import net.minecraft.commands.CommandSourceStack
import net.minecraft.commands.Commands.*

object Timer : BrigadierCommand {
    override fun register(dispatcher: CommandDispatcher<CommandSourceStack>) {
        dispatcher.register(literal("timer").requiresOp().then(argument("time", IntegerArgumentType.integer()).executes { ctx ->
            GameData.timerMs = System.currentTimeMillis() + IntegerArgumentType.getInteger(ctx, "time")
            1
        }).then(literal("clear").executes { ctx ->
            GameData.timerMs = -1L
            1
        }))
    }
}